src/test/java/StringTest.java

```diff
@@ -0,0 +1,73 @@
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("String 클래스의 ")
+public class StringTest {
+
+    @Nested
+    @DisplayName("split 메소드를 사용하여  ")
+    class SplitTest {
+
+        private final String delimiterComma = ",";
+
+        @Test
+        @DisplayName("콤마로 분리된 문자열 1,2 를 콤마를 중심으로 문자열 1과 2로 쪼갠다")
+        void givenStringWithComma_whenSplit_thenReturnParts() {
+            String strWithComma = "1,2";
+
+            String[] parts = strWithComma.split(delimiterComma);
+
+            Assertions.assertThat(parts).hasSize(2);
+            Assertions.assertThat(parts[0]).isEqualTo("1");
```

Comment:
특정 값만을 가지고 있는지 검증하는 메서드를 사용해보면 어떨까요?
```suggestion
            assertThat(parts).containsExactly("1", "2");
```